include/revng/ABI/CallingConventionTrait.h

```diff
@@ -0,0 +1,240 @@
+#pragma once
```

Comment:
`CallingConventionTrait.h` -> `ABITrait.h`? Even if right this is focused on the calling convention only, we might add other stuff concerning the ABI (struct alignment, etc...).

Reply:
Done!